src/Actions/Admin/Find.cs

```diff
@@ -0,0 +1,133 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="Find.cs" company="WheelMUD Development Team">
+//   Copyright (c) WheelMUD Development Team.  See LICENSE.txt.  This file is
+//   subject to the Microsoft Public License.  All other rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using WheelMUD.Core;
+
+namespace WheelMUD.Actions
+{
+    /// <summary>A command that allows an admin to locate an entity.</summary>
+    [CoreExports.GameAction(0)]
+    [ActionPrimaryAlias("find", CommandCategory.Builder)]
+    [ActionDescription("Finds Things that match the specified argument.")]
+    [ActionSecurity(SecurityRole.minorBuilder)]
+    public class Find : GameAction
+    {
+        /// <summary>List of reusable guards which must be passed before action requests may proceed to execution.</summary>
+        private static readonly List<CommonGuards> ActionGuards = new List<CommonGuards>
+        {
+            CommonGuards.RequiresAtLeastOneArgument,
+            CommonGuards.InitiatorMustBeAPlayer
+        };
+
+        private static readonly ThingManager _thingManager = ThingManager.Instance;
+
+        public override void Execute(ActionInput actionInput)
+        {
+            // To determine what type of find to use.
+            var findType = actionInput.Params[0];
+            // Find params from rest of Params array.
+            var findParams = actionInput.Params.Skip(1).ToArray();
+            // Set this to true if we find anyThing.
+            var thingFound = false;
+
+            if ("id".Contains(findType.ToLower()))
+            {
+                if (TryFindThingByID(string.Join(" ", findParams), out var thing))
+                {
+                    WriteOneThing(actionInput, thing, true);
+                    thingFound = true;
+                }
+            }
+            else if ("keyword".Contains(findType.ToLower()))
```

Comment:
These probably aren't doing what you intended.  This searches for a substring.  E.G.
`"id".Contains("d")` is true so would go into the first case, `"keyword".Contains("y")` would be true, etc.

You might try:
 `if ("id".Equals(findType, StringComparison.OrdinalIgnoreCase) || "i".Equals(findType, StringComparison.OrdinalIgnoreCase)`
 and so on to take `id` or `i`, `keyword` or `k`, or `name` or `n`?

However...
Strategically, I think we could make this more user friendly than needing the type argument at all.  Considering that IDs should always be a multi-part strings like `namespace/id`, and keywords shouldn't include `/` at all, I think we could basically:
* Try to find by exact ID.
* If not found, try to find matches by keyword.
* If still none found, try to find matches by name.

Reply:
Oohh, I think I understand it now, it would've returned true for any char in any order right. 
Your suggestion seems like it would be closer to what I intended yeah, or maybe I coul've used `StartsWith()`?

Either way I removed it as per suggestion, unneeded extra parameter indeed.